src/_locales/tr/messages.json

```diff
@@ -0,0 +1,442 @@
+    },
+    "intro_green": {
+        "message": "Yeşil",
+        "description": ""
+    },
+    "intro_no_tracker": {
+        "message": "Privacy Badger'ın bunun takipçi olmadığına inandığı anlamına gelir. Tarayıcınızın araç çubuğunda Privacy Badger ikonuna tıklayarak otomatik engelleme ayarlarını değiştirebilirsiniz. Veya Privacy Badger otomatik olarak web takipçilerini bulup birer birer yerken huzur içinde gezinmeye devam edebilirsiniz.",
+        "description": ""
+    },
+    "options_domain_list_no_trackers": {
+        "message": "Privacy Badger hiçbir <a target='_blank' tabindex=-1 title='i18n_what_is_a_tracker' class='tooltip' href='https://www.eff.org/privacybadger#faq-What-is-a-third-party-tracker?'>takipçi adres</a> algılamadı. Gezinmeye devam!",
+        "description": "Shown on the Tracking Domains tab on the options page for new and recent installations."
+    },
+    "slideshow_15": {
+        "message": " Elbette bütün seçenekleri yalnız bırakabilirsiniz, ben de bulduğum takipçileri sessizce yerim.",
+        "description": ""
+    },
+    "report_input_label": {
+        "message": "Açıklama",
+        "description": ""
+    },
+    "non_tracker": {
+        "message": "Aşağıdaki adresler sizi takip etmiyor",
+        "description": ""
+    },
+    "slideshow_14": {
+        "message": " Diğer seçenekleri değiştirmek isterseniz, bu dişli ikonuna tıklayabilirsiniz.",
+        "description": ""
+    },
+    "popup_options_button": {
+        "message": "Ayarlar",
+        "description": ""
+    },
+    "slideshow_6": {
+        "message": " Eğer kaydırıcı sarıysa, bu takipçiyi kısmen engellediğim anlamına gelir. Çerezleri yiyip sizi izlemesini engellemek için elimden geleni yapacağım, ama onu tamamen engellemek sayfayı kullanılamaz hale getirebilir.",
+        "description": ""
+    },
+    "slideshow_16": {
+        "message": " Gizliliğinizi korumaya yönelik kolay ve önemli bir adım attığınız için teşekkürler! Eğer tekrar yardıma ihtiyacınız olursa buraya tıklayabilirsiniz, ve dünyanın online takibe karşı durduğunuzu bilmesini istiyorsanız aşağıya tıklayabilirsiniz.",
+        "description": ""
+    },
+    "share_button_title_facebook": {
+        "message": "Facebook'ta Paylaş",
+        "description": "Tooltips that come up when you hover over the social sharing buttons on the introductory slideshow page."
+    },
+    "description": {
+        "message": "Privacy Badger siz internette dolaşırken sizi takipçilerden korur!",
+        "description": ""
+    },
+    "popup_noblocked": {
+        "message": "Neden hiçbir şey engellenmemiş?",
+        "description": ""
+    },
+    "report_fail": {
+        "message": "Tüh. Birşeyler ters gitti.",
+        "description": ""
+    },
+    "slideshow_10": {
+        "message": " Eğer bir takipçi hakkında yanlış karar verdiğimi düşünüyorsanız, kaydırıcıyı kırmızıya götürerek onu tamamen, sarıya götürerek kısmen engelleyebilir, ve yeşile götürerek ona izin verebilirsiniz.",
```

Comment:
It might just be my translation tool (apologies if false alarm), but this might be inaccurate ... Does this say "move to red to block entirely, move to yellow to partially block, move to green to allow"?